src/Operation/EstimatedDocumentCount.php

```diff
@@ -111,7 +110,7 @@ public function __construct($databaseName, $collectionName, array $options = [])
      */
     public function execute(Server $server)
     {
-        $command = $this->createCommand($server);
+        $command = $this->createCount();
```

Comment:
The `if ($command instanceof Aggregate)` conditional below is dead code. Please replace the entire body of this method with:

```suggestion
        return $this->createCount()->execute($server);
```

Remember to re-run `phpcbf` after doing so, as some imports may need to be removed (e.g. CommandException).